### PR TITLE
fix data process to predict

### PR DESCRIPTION
**Pull request by rickycao-qy**
Since we have changed how to data process works, need to change predict script accordingly

### CODE REVIEW

File: packages/daemon/templates/node/predict.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ async function predict(data) {
   if (!model) {
     model = await modelDefine;
   }
-  const sample = { data, label: null };
+  let sample = { data, label: null };
   if (typeof dataProcessModule === 'function') {
-    await dataProcessModule(sample, {}, JSON.parse(pipeline.dataProcessParams));
+    sample = await dataProcessModule(sample, {}, JSON.parse(pipeline.dataProcessParams));
```
Comment (yorkie) [inline packages/daemon/templates/node/predict.js:34]: Dos the __data-process__ plugin return `Sample` now?

Comment (rickycao-qy) [reply packages/daemon/templates/node/predict.js:34]: Exactly, data-process and global-data-process will not modify original data.

Comment (yorkie) [reply packages/daemon/templates/node/predict.js:34]: See https://github.com/alibaba/pipcook/blob/master/docs/spec/plugin.md#plugin-category, this needs an update IMO.

Comment (rickycao-qy) [reply packages/daemon/templates/node/predict.js:34]: Done ~~

**Review by yorkie — APPROVED** [review]

**Pull request merged**